eta/core/data.py

```diff
@@ -531,6 +532,78 @@ def remove_schema(self):
         '''Removes the enforced schema from the container.'''
         self.schema = None
 
+    def has_attr_with_name(self, name):
+        '''Returns whether or not the container contains an Attribute with the
+        give name.
+
+        Args:
+            name: the Attribute name
+
+        Returns:
+            boolean indicating whether or not the container contains an
+                Attribute with the given name
+        '''
+        return bool(self.get_attrs_with_name(name))
+
+    def get_attrs_with_name(self, name):
+        '''Get a list of all attributes with a given name
```

Comment:
prefer not to say "list" here because you're actually returning a `Container`.

Really glad that the suggestion was already made to return a `Container`, not a `list`. `list`s of `Attribute`s should never be created. `AttributeContainer` is a fancy way to store lists of `Attribute`s that provides more convenience methods